xml/System.Reflection.Emit/AssemblyBuilder.xml

```diff
@@ -2532,9 +2532,17 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates whether this dynamic assembly is held in a collectible <see cref="T:System.Runtime.Loader.AssemblyLoadContext" />.</summary>
+        <value>
+          <see langword="true" /> if this dynamic assembly is held in a collectible <see cref="T:System.Runtime.Loader.AssemblyLoadContext" />; otherwise, <see langword="false" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ For more information, see [How to use and debug assembly unloadability in .NET Core](/dotnet/standard/assembly/unloadability).
```

Comment:
I assume this link renders correctly as `https://docs.microsoft.com/dotnet/standard/assembly/unloadability` ?

Reply:
I think it will render correctly.

Here's the preview page: https://review.docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.assemblybuilder.iscollectible?view=net-6.0&branch=pr-en-us-7016

Go down to remarks and hover over the hyperlink. Notice it's prefixed with `https://review.docs.microsoft.com/` and if you click it, it does not exist.

Once it's published, it should remove the `review` word, and the link should work.